pkg/reader/usdc_reader.go

```diff
@@ -91,6 +91,11 @@ func NewUSDCMessageReader(
 		}
 		switch family {
 		case sel.FamilyEVM:
+			contractReader, ok := contractReaders[chainSelector]
+			if !ok {
+				lggr.Warnf("chain reader is missing for chain %d, skipping", chainSelector)
```

Comment:
Shouldn't that log on error level? It might be related to some underlying misconfiguration